packages/test/test-end-to-end-tests/src/test/container.spec.ts

```diff
@@ -137,6 +145,18 @@ describeNoCompat("Container", (getTestObjectProvider) => {
 			"Client details should be set with interactive as true",
 		);
 	});
+	it.only("Delta manager receives pong event", async () => {
+		if (provider.driver.type !== "local" && provider.driver.type !== "odsp") {
+			this.skip();
+		}
+		const container = await createConnectedContainer();
+		// let initial registration of pong event happen
+		clock.tick(60000);
+		// real pong events will take at least a minute to fire in real time, so exit test when we receive the real one.
+		container.deltaManager.on("pong", () => {
+			process.exit();
```

Comment:
process.exit() is not the right solution here - it ensures that none of the tests below will run.
I think you want something like this:

```ts
await new Promise((accept) => {
    container.deltaManager.on("pong", accept);
}
```